### PR TITLE
docs: reformat exit

### PR DESCRIPTION
**Pull request by Yongxuanzhang**
reformat exit.rst to follow the guide

- [ ] wait for new layout

### CODE REVIEW

File: chapters/cli/exit.rst
Comment (FionnD) [inline chapters/cli/exit.rst]: Nothing wrong with this, but I think we can use stronger verbs. (https://developers.google.com/tech-writing/one/clear-sentences#choose_strong_verbs). 

What do you think this re-work? 

In this section, you will learn best practices to shut down a Flow and exit Jina correctly.

File: chapters/cli/exit.rst
Comment (FionnD) [inline chapters/cli/exit.rst]: I think we can shorten this to: 

There are several methods to exit Jina gracefully. 

Or do you think we are losing important information?

**Review by FionnD — CHANGES_REQUESTED** [review]
Looks good to me :) 

Two minor grammar suggestions. 

Once a Dev has reviewed you can merge

File: chapters/cli/exit.rst
Comment (Yongxuanzhang) [reply chapters/cli/exit.rst]: Thanks! Much better!

File: chapters/cli/exit.rst
Comment (Yongxuanzhang) [reply chapters/cli/exit.rst]: Thanks! Revised in the latest commit

**Review by FionnD — APPROVED** [review]
Looks great to me!

File: chapters/cli/exit.rst
Comment (CatStark) [inline chapters/cli/exit.rst]: Typo in "your" it should be "you" right?

File: chapters/cli/exit.rst
Comment (CatStark) [inline chapters/cli/exit.rst:1]: Could you also add a table of contents? :)
.. contents:: Table of Contents
    :depth: 2

File: chapters/cli/exit.rst
Comment (Yongxuanzhang) [reply chapters/cli/exit.rst]: ✔️

File: chapters/cli/exit.rst
Comment (Yongxuanzhang) [reply chapters/cli/exit.rst:1]: ✔️

**Comment by bwanglzu** [comment]
@Yongxuanzhang do not need to wait for new layout, after being reviewed, merge and we'll re-oranigze it.

**Pull request merged**
